core/src/main/scala/kafka/server/AbstractFetcherThread.scala

```diff
@@ -145,10 +145,9 @@ abstract class AbstractFetcherThread(name: String,
                 case Errors.NONE =>
                   try {
                     val messages = partitionData.toByteBufferMessageSet
+                    val validBytes = messages.validBytes
```

Comment:
This is only used a few lines later. Is there a reason why we are assigning the val here?